react/components/BuyButton/index.js

```diff
@@ -89,26 +90,27 @@ export class BuyButton extends Component {
         () => {
           this.toastMessage(false)
         }
-      )
+      ).finally(() => {
+        this.setState({ isAddingToCart: false })
+      })
   }
 
   render() {
     const { children, skuItems, available } = this.props
     const loading = this.state.isLoading || !skuItems
+    const isAddingToCart = this.state.isAddingToCart
```

Comment:
change it to `const { isAddingToCart } = this.state`. It's cleaner and will help in future additions